elements/pfe-sass/variables/_maps.scss

```diff
@@ -53,6 +54,9 @@ $pfe-colors: (
   ui-disabled--text:                  $pfe-color--ui-disabled--text,
   ui-disabled--text--hover:           $pfe-color--ui-disabled--text--hover,
 
+  ui-tab--highlight-color:            $pfelements--red,
```

Comment:
Can we use accent instead of adding a new custom variable for this?

Reply:
+1

Reply:
@castastrophe removed this variable.